Source/ACE.Server/Command/Handlers/AdminCommands.cs

```diff
@@ -1584,59 +1584,268 @@ public static void HandleFumble(Session session, params string[] parameters)
         }
 
         // god
-        [CommandHandler("god", AccessLevel.Sentinel, CommandHandlerFlag.RequiresWorld, 0)]
+        [CommandHandler("god", AccessLevel.Sentinel, CommandHandlerFlag.RequiresWorld, 0,
+            "Turns current character into a god!",
+            "Sets attributes and skills to higher than max levels.\n"
+            + "To return to a mortal state, use the /ungod command.\n"
+            + "Use the /god command with caution. While unlikely, there is a possibility that the character that runs the command will not be able to return to normal or will end up in a state that is unrecoverable.")]
         public static void HandleGod(Session session, params string[] parameters)
         {
             // @god - Sets your own stats to a godly level.
+            // need to save stats so that we can return with /ungod
+            var biotas = new Collection<(Biota biota, ReaderWriterLockSlim rwLock)>();
```

Comment:
This looks like it should work, and is probably actually good way to do it, just caught me off guard since it deviates from the standard session.Player.SaveBiotaToDatabase() (which doesn't have a callback)

I would have probably cheaped out here and just called session.Player.SaveBiotaToDatabase(), and then put the Ungod call in an ActionChain that runs a small amount of time later, but this is probably actually technically better

Now that I think about it though... why does the player need to be explicitly saved before entering god mode?

Reply:
I figured it was a good idea to save a snapshot to db before tinkering with all of the stats.